tools/scoring/score/dimensions/entity_connection_identification.py

```diff
@@ -27,41 +27,35 @@ class EntityConnectionIdentification(Dimension):
   Quantifies whether connections between entities were
   correctly and completely defined in the proposed file.
   """
-
-  # TODO: Figure out how to elegantly implement "facilities"
-  # and "equipment" categories given current object model
-  def __init__(self, *, deserialized_files: DeserializedFilesDict):
-    super().__init__(deserialized_files=deserialized_files)
-
-    proposed, solution = map(deserialized_files.get, (PROPOSED, SOLUTION))
-
-    # Isolate the connections from each dictionary of entities
-    solution_connections = [
-        tup for tup in (((cloud_device_id, connection)
-                         for connection in entity.connections)
-                        for cloud_device_id, entity in solution.items()
-                        if entity.connections is not None) for tup in tup
-    ]
-    proposed_connections = [
+  def _isolate_connections(self, file: DeserializedFile):
+    return [
         tup for tup in (((cloud_device_id, connection)
                          for connection in entity.connections)
-                        for cloud_device_id, entity in proposed.items()
+                        for cloud_device_id, entity in file.items()
                         if entity.connections is not None) for tup in tup
     ]
 
-    fstring = (lambda target, connection:
-               f'{target} {connection.ctype} {connection.source}')
-
-    # Condense them into sets of strings for easy comparison using intersection
-    solution_connections_condensed = set([
-        fstring(target, connection)
-        for target, connection in solution_connections
-    ])
-    proposed_connections_condensed = set([
-        fstring(target, connection)
-        for target, connection in proposed_connections
+  def _condense_connections(self, connections):
+    """ Condense connections into sets of strings
+    for easy comparison using intersection """
+    return set([
+        f'{target} {connection.ctype} {connection.source}'
+        for target, connection in connections
     ])
 
+  # TODO: Figure out how to elegantly implement "facilities"
+  # and "equipment" categories given current object model
+  def evaluate(self):
```

Comment:
can you please add docstring

Reply:
85c589b8964d292819a342cd0c3dd7006e3705d9

Reply:
85c589b8964d292819a342cd0c3dd7006e3705d9